utils/data.py

```
@@ -0,0 +1,25 @@
+import json
+import os
+from typing import List, Dict, Any
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)